doc/configuration.rst

```diff
@@ -58,6 +58,8 @@ file:
 - ``log_level`` (:ref:`log_level`)
 - ``capture_repr`` and ``ignore_repr_types`` (:ref:`capture_repr`)
 - ``nested_sections`` (:ref:`nested_sections`)
+- ``missing_doc_ignore`` (:ref:`missing_doc_ignore`)
```

Comment:
nitpick, I wonder if something like 'api_usage_ignore' might be a better config name as it makes it clearer that it is related to the `show_api_usage` config. I feel 'missing_doc_ignore' may be a little ambiguous.